### DISCUSSION TITLE
Support for Dreamehome App

### DISCUSSION THREAD

**Discussion by autumn2go** [merdok/homebridge-miot#461] [Ideas]
Hey!

Is there any support or implementation planned for the Dreamehome app?

I had always integrated my vacuum robots in the Xiaomi app, but with the L10S Ultra I switched to the Dreamehome app. 

Thanks a lot!

**Comment by merdok** [comment]
What do you mean exactly with support for the Dreamehome app?

**Comment by autumn2go** [comment]
Currently, you can only use the devices with the plugin, if you use the Xiaomi home app.

I ask for an implementation where the token can also be read from the Dreamehome app and integrated into the plugin.

Thanks for the quick feedback.

**Reply by merdok** [reply]
As far as I know this is currently not possible. I do not know at this point of any way on how to retrieve the token from the Dreamehome app...

**Reply by autumn2go** [reply]
I did not know that. So I have no other option than to add it in the Xioami app and use it there if I want to use the plugin?

**Reply by merdok** [reply]
I know that people somehow retrieve the tokens for the dreame devices, not sure if they do it over the xiaomi app or there is another way... the Xiaomi way is definitely the easiest as it can be then retrieved from the cloud.

**Comment by kolorfilm** [comment]
I’m also very interested in an implementation. The dreamehome app has firmware updates much faster and a few more features.

It seems the devs from Home Assistent made it possible to use the Dreamehome app: https://github.com/Tasshack/dreame-vacuum/releases

It would be great if this works also for Homebridge.

**Reply by merdok** [reply]
Thanks for the info. I do not see anything special at first as the requests seems to go against the customized endpoints as this plugins does... would need to investigate further when I have more time...
